tests/utils/channel_utils.test.jsx

```diff
@@ -1,27 +1,58 @@
 // Copyright (c) 2016-present Mattermost, Inc. All Rights Reserved.
 // See License.txt for license information.
+import {GeneralTypes} from 'mattermost-redux/action_types';
 
 import * as Utils from 'utils/channel_utils.jsx';
 import Constants from 'utils/constants.jsx';
+import store from 'stores/redux_store.jsx';
 
 describe('Channel Utils', () => {
+    afterEach(() => {
+        store.dispatch({
```

Comment:
I found it impossible to mock the store except when used by components and you can inject a mock version via the Provider. This approach effectively allows me to mutate the store at will.